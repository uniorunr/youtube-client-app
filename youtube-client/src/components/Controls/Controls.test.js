import controlsComponent from './Controls';

describe('controlsComponent', () => {
  const main = document.createElement('main');
  document.body.appendChild(main);

  it('should be instance of Function', () => {
    expect(controlsComponent).toBeInstanceOf(Function);
  });

  it('should be rendered correctly', () => {
    controlsComponent(main);
    expect(document.body.innerHTML).toMatchSnapshot();
  });
});
