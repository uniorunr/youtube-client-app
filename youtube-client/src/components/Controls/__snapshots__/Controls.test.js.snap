// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`controlsComponent should be rendered correctly if controls already added to the page 1`] = `"<main><section class=\\"controls-section\\"><div class=\\"controls\\"><button class=\\"controls__button controls__button_prev controls__button_state_disabled\\" type=\\"button\\" data-page=\\"0\\"><i class=\\"fas fa-caret-left\\"></i><span class=\\"controls__prev-control-content\\">0</span></button><button class=\\"controls__button controls__button_curr\\" type=\\"button\\" data-page=\\"1\\">1</button><button class=\\"controls__button controls__button_next\\" type=\\"button\\" data-page=\\"2\\"><span class=\\"controls__next-control-content\\">2</span><i class=\\"fas fa-caret-right\\"></i></button></div></section></main>"`;

exports[`controlsComponent should be rendered correctly if controls aren't added to the page 1`] = `"<main><section class=\\"controls-section\\"><div class=\\"controls\\"><button class=\\"controls__button controls__button_prev controls__button_state_disabled\\" type=\\"button\\" data-page=\\"0\\"><i class=\\"fas fa-caret-left\\"></i><span class=\\"controls__prev-control-content\\">0</span></button><button class=\\"controls__button controls__button_curr\\" type=\\"button\\" data-page=\\"1\\">1</button><button class=\\"controls__button controls__button_next\\" type=\\"button\\" data-page=\\"2\\"><span class=\\"controls__next-control-content\\">2</span><i class=\\"fas fa-caret-right\\"></i></button></div></section></main>"`;
